src/main/jhipster/src/main/webapp/app/germplasm-details/genotyping/genotyping-pane.component.ts

```diff
@@ -169,16 +172,24 @@ export class GenotypingPaneComponent implements OnInit {
     }
 
     selectVariantsetOnChange() {
-        this.genotypingBrapiService.searchCallsets({
-            variantSetDbIds: [this.selectedVariantSet.variantSetDbId],
-            germplasmDbIds: [this.genotypingGermplasm.germplasmDbId]
-        }).subscribe((brapiResponse) => {
-            if (brapiResponse && brapiResponse.result.data.length) {
-                this.genotypingCallSet = brapiResponse.result.data[0];
-                this.loadGenotypingCalls();
-            } else {
-                this.alertService.error('genotyping.no.genotyping.callsets.found');
-            }
+        const exportFlapjackRequest = new ExportFlapjackRequest([], [], 'FLAPJACK', [this.germplasmSearchValue], true,
+            100, this.selectedVariantSet.referenceSetDbId);
+        this.genotypingBrapiService.exportFlapjack(exportFlapjackRequest).subscribe((response) => {
```

Comment:
@mderamos  exportingFlapjack Gigwa data into Flapjack format can take a while if the data is big. It would be nice to have a progress UI that indicates the export is in progress.

Reply:
Hi @abatac , there's a progress bar for download progress. ![image](https://user-images.githubusercontent.com/13634210/191174615-7f09e133-60a5-404c-93ee-7405a4e6c09e.png)